modules/introduction/partials/new-features-76_2.adoc

```diff
@@ -9,3 +9,7 @@
 ** xref:rest-api:backup-get-task-info.adoc[`/api/v1/cluster/self/repository/{repo-state}/{task-name}/taskHistory`]
 ** xref:rest-api:backup-get-plan-info.adoc[`/api/v1/plan/`]
 
+* You can now set the number of threads each Backup Service node uses when backing up data.
+For example, if you find backups cause performance issues on your cluster, you can reduce the number of threads the Backup Service uses.
+Reducing the number of threads also reduces the number of concurrent client connections the Backup Service makes to retrieve data.
```

Comment:
That's a good point. However, I think it would be better to add that to the Thread Usage section in the Backup Service instead of the What's New section. I've made that update and will rebuild the preview site.